announcing-establishment-of-japan-office.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Announcing Establishment of Japan Office
+author: ['PingCAP']
+date: 2021-04-22
+summary: In order to better serve the Japan market and support its growth, PingCAP formally established a new office in Tokyo, Japan.
+image: /images/blog/announcing-establishment-of-japan-office.png
+tags: ['Announcement']
+categories: ['Company']
+press_release: true
+---
+
+**Author:** PingCAP
+
+**Editors:** [Calvin Weng](https://github.com/dcalvin), [Queeny Jin](https://github.com/queenypingcap)
+
+![Announcing New Operations in Japan](media/announcing-establishment-of-japan-office.png)
+
+TOKYO, JAPAN, APRIL 22, 2021 - In order to better serve the Japan market and support its growth, PingCAP, the enterprise-grade open-source distributed database provider, announced that the Japan office is formally established in Tokyo in April 2021.
+
+"PingCAP’s vision is to make growth easier for global innovation leaders," Says Max Liu, CEO at PingCAP, "With a rising demand for open source infrastructure solutions like TiDB, Japan is an important market for us. We are excited about our new operations in Japan and are committed to serving the region’s fast growing customers with their digital transformation journey while contributing to the local open source market.”
+
+PingCAP’s flagship product, TiDB, is an open-source hybrid transactional and analytical (HTAP) database featuring horizontal scalability, strong consistency, and high availability with MySQL compatibility. Over the last 5 years, TiDB has been adopted and battle-tested by more than 1,500 companies globally across multiple industries such as finance, e-commerce, payment, online video, gaming, logistics, media, manufacturing, and travel.
+
+In June 2020, PingCAP announced the beta release of [TiDB Cloud](https://en.pingcap.com/products/tidbcloud/), the fully managed TiDB service on AWS and GCP, which reduces the operational cost of using TiDB while allowing customers to focus on their applications development and not the complexities of their database.
+
+PingCAP has adopted open source as a core strategy of developing and promoting the next-gen database, especially in the context of cloud-native computing. TiDB has been open source since day one with its two sibling projects: TiKV, a distributed key-value storage, and Chaos Mesh®, a cloud-native chaos engineering platform. Both [TiKV](https//tikv.org/) and [Chaos Mesh®](https//www.chaos-mesh.org/) have been widely adopted around the world and are in [Cloud Native Computing Foundation](https://cncf.io) (CNCF) Graduated and Sandbox stages, respectively. In addition, PingCAP has been among the top 10 of the CNCF global contribution ranking list for the past 2 years in a row.
```

Comment:
```suggestion
PingCAP has adopted open source as a core strategy of developing and promoting the next-gen database, especially in the context of cloud-native computing. TiDB has been open source since day one with its two sibling projects: TiKV, a distributed key-value storage, and Chaos Mesh®, a cloud-native chaos engineering platform. Both [TiKV](https.//tikv.org/) and [Chaos Mesh®](https.//www.chaos-mesh.org/) have been widely adopted around the world and are in [Cloud Native Computing Foundation](https://cncf.io) (CNCF) Graduated and Sandbox stages, respectively. In addition, PingCAP has been among the top 10 of the CNCF global contribution ranking list for the past 2 years in a row.
```